confidant/services/jwkmanager.py

```diff
@@ -67,17 +69,20 @@ def get_jwt(self, kid: str, payload: dict,
             'exp': expiry,
         })
 
-        token = jwt.encode(
-            payload=payload,
-            headers={'kid': kid},
-            key=key.export_to_pem(private_key=True, password=None),
-            algorithm=algorithm,
-        )
+        with stats.timer('get_jwt.encode'):
+            # XXX: TODO: cache export_to_pem
+            token = jwt.encode(
+                payload=payload,
+                headers={'kid': kid},
+                key=key.export_to_pem(private_key=True, password=None),
+                algorithm=algorithm,
+            )
 
         self._token_cache[kid][user] = {
             'expiry': expiry,
             'token': token
         }
+        stats.incr('get_jwt.cache.miss')
```

Comment:
do we want to also record this is caching is disabled?

Reply:
hrmm, yeah that's a good point.  i've renamed this to `get_jwt.create`